parlemonde/clap
# fix user_feedbacks

### Motivation


Fix bugs and add new features/improvements 

### Changes


- https://trello.com/c/8p3AX29R/29-bug-configuration-du-titre-de-la-s%C3%A9quence
- https://trello.com/c/WRjOfEEs/30-bug-g%C3%A9rer-les-textes-longs
- https://trello.com/c/kUj0quBD/31-bug-mise-en-forme-diff%C3%A9rente
- https://trello.com/c/6T7l5qOU/34-modification-phrase-de-fin
- https://trello.com/c/7Gis0OZv/38-bug-recadrage-des-images
- https://trello.com/c/BBicYj7H/27-bug-r%C3%A9organisation-des-s%C3%A9quences
- https://trello.com/c/2zkgYBup/42-modification-du-comportement-de-la-souris
- https://trello.com/c/V1J9Rvyd/32-bug-suppression-du-son-import%C3%A9
- https://trello.com/c/hdhmcqFy/36-feature-format-m4a
- https://trello.com/c/0S613fk2/35-feature-r%C3%A9agencement-plan
- https://trello.com/c/DNlPKJBZ/28-feature-am%C3%A9liorer-la-liste-des-utilisateurs
- https://trello.com/c/ZC33oWjB/46-bug-erreur-500-quand-on-refresh-la-page-de-gestions-des-utilisateurs-c%C3%B4t%C3%A9-admin
- https://trello.com/c/eGD2jOH1/43-bug-bouton-play

### Test


- Try to change the position of sequence's title (left, center, right)
- Try to add a long text in the sequence's title, it shouldn't overflow
- Background should be white when you download the movie 
- Try to upload a wide image, you should be able to select the whole image 
- Try to change sequence's order, error message shouldn't be displayed anymore
- Try to upload an mp3 file, the sound should be persisted even if you change page or refresh
- Try to upload a M4A file 
- Try to change plan's order (design has been changed on purpose)
- New columns have been added in users table (admin)
- Try to refresh admin's users management page, you shouldn't have error 500 anymore
- On create last step the play button button should not be over modal anymore
